src/components/Forms/SignInForm/style.js

```diff
@@ -0,0 +1,38 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    height: '100vh',
+  },
+  image: {
+    backgroundImage: 'url(/images/goodwilllogo.png)',
+    backgroundRepeat: 'no-repeat',
+    backgroundColor:
+      theme.palette.type === 'dark' ? theme.palette.grey[900] : theme.palette.grey[50],
```

Comment:
If you are NOT planning to have multiple themes, you may pick a color and remove this ternary statement.

Reply:
will be done offline